docs/digger/admin/use-http-proxy.asciidoc

```diff
@@ -1,3 +1,5 @@
+== HTTP Proxy Support
+
 === Configuration Options
 
 It is possible to run AeroGear Digger behind a HTTP proxy server. To configure the proxy server, you need to add the following Ansible Variables:
```

Comment:
We also need to include proxy_url as this is used in a number of [places in the installer](https://github.com/aerogear/digger-installer/search?utf8=%E2%9C%93&q=proxy_url&type=)